### ISSUE TITLE
Discovery for Wish Lists in Venia (MVP+)

### ISSUE THREAD

**Issue by ericerway** [magento-research/pwa-studio#467]
## This issue is for the following packages:

[X] `venia-concept`
[ ] `pwa-buildpack`
[ ] `peregrine`
[ ] `pwa-module`
[ ] `pwa-devdocs`

## Description

![image](https://user-images.githubusercontent.com/2686596/48219721-d8dd3800-e352-11e8-9a3b-899a7ac8c687.png)

![image](https://user-images.githubusercontent.com/2686596/48220280-8735ad00-e354-11e8-8af8-a82478d53e82.png)

As a shopper I want to add a product from Venia to a wish list (single) so that I can make a purchase in the future.

- Reinstate Add to Wishlist from Product Details (all products)
- Enable Wishlist for Venia customers and show confirmation (temporary)
- Verify updates in Magento 2 admin by customer (Wish Lists)

#### Assumptions
- Authentication for Venia complete via Create Account and login experience
- Account experience will be handled as a separate story; similar to Purchase History; this will handle view, management, sharing and adding all to cart
- Multiple wish lists (Commerce/EE) will be handled in the future
- GraphQL usage in 2.3.0-develop (post-MVP) preferred

## Tasks
[ ] Discovery for GraphQL/REST API coverage around customer wish list (single); including versions for 2.3.0-develop (as needed)
[ ] Implement basic wish list functionality (with placeholder confirmation)
[ ] Verify that wish list has been updated (by customer) in Magento 2 admin
[ ] Testing and documentation (https://github.com/magento-research/pwa-studio/wiki/Documentation-Guidelines)

**Comment by ericerway** [comment]
Kamino cloned this issue to [magento-research/pwa-studio](https://github.com/magento-research/pwa-studio/issues/476)

**Comment by AlexeyKaryka** [comment]
Tasks:

- [ ] Create and set up GraphQL mutation schema for creating new wishlist items on the backend side(at now there is only schema for retrieving existing wishlist items)

- [ ] Create GraphQL mutation schema on the frontend side

- [ ] Reinstate "Add to wishlist button" and set up it for processing graphql requests

- [ ] Implement temporary confirmation of successful adding to a wishlist

**Comment by AlexeyKaryka** [comment]
Blocked by https://github.com/magento/magento2/issues/19410
